src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -129,12 +129,14 @@
     <Error Text="The AccountKey property must be set." Condition="'$(AccountKey)' == ''" />
     <Error Text="The ManifestName property must be set." Condition="'$(ManifestName)' == ''" />
     <Error Text="The ManifestBuildId property must be set." Condition="'$(ManifestBuildId)' == ''" />
+    <Error Text="The IsStable property must be set" Condition="'$(IsStable)' == ''" />
 
     <PropertyGroup>
       <ManifestTempDir Condition="'$(ManifestTempDir)' == ''">$(BaseIntermediateOutputPath)manifest-temp/</ManifestTempDir>
       <ManifestDownloadDir Condition="'$(ManifestDownloadDir)' == ''">$(ManifestTempDir)downloads/</ManifestDownloadDir>
       <ManifestFile Condition="'$(ManifestFile)' == ''">$(ManifestTempDir)build.xml</ManifestFile>
       <ManifestSummaryFile Condition="'$(ManifestSummaryFile)' == ''">$(ManifestTempDir)README.md</ManifestSummaryFile>
+      <VersionStamp Condition="'$(VersionStamp)' == ''">N/A</VersionStamp>
```

Comment:
I think we shouldn't do this to avoid the special string. (A bit of miscommunication made me think I wanted this, but I have to write a new updater for this approach so null/empty will work.)